k8s-network-policies/only-egress-to-kubeapiserver.yaml

```diff
@@ -1,3 +1,7 @@
+# This policy will block all egress traffic not destined for the kube-apiserver
```

Comment:
Added more info on why this policy would be used.

Reply:
I'd suggest changing from `the namespace where it's applied` to `default namespace`, since that's how the policy is currently written.

Also - I think it would be clearer to write something like `This policy allows traffic to the Kubernetes API server only and block all other egress traffic...` 

something like that. That being said - looks good to me.